app/controllers/admin/items_controller.rb

```
@@ -6,4 +6,12 @@ def index
     item_search_setup
   end
 
+  def destroy
+    @item = JupiterCore::LockedLdpObject.find(params[:id], types: [Item, Thesis])
+    authorize @item
+    @item.unlock_and_fetch_ldp_object(&:destroy!)
```

Considering this is activefedora and weird things happen...

Should we handle the fail case here? What if delete fails? 

(Assuming activefedora works like activerecord...which is a big assumption) `destroy!` (bang version) will raise an exception (in activerecord it raises ActiveRecord::RecordNotDestroyed). Should we be using the non bang version? `destroy` and just let it fail quietly?

Or should we handle this gracefully and return a decent error to the user?
```ruby
if  @item.unlock_and_fetch_ldp_object(&:destroy)
  redirect_back(fallback_location: root_path, notice: t('.successful_deletion'))
else
 redirect_back(fallback_location: root_path, notice: t('.failed_deletion'))
end
```

---

If we can't trust ActiveFedora to work like ActiveRecord then maybe this would be better?
```
begin
      @item.unlock_and_fetch_ldp_object(&:destroy!)
      flash[:notice] = t('.deleted')
    rescue StandardError => e
      flash[:alert] = t('.failed')
      Rollbar.error("Error deleting #{@item.id}", e)
    end
```
That would catch any errors from the unlock_and_fetch_ldp_object step (like LDP:Gone) and anything else that Fedora/ActiveFedora might come up with. I was looking for some documentation about destroy vs destroy! and found that even the non bang version [doesn't always work as expected](https://github.com/rails/rails/issues/18056).